app/src/main/res/layout/item_content_cart.xml

```diff
@@ -61,64 +63,61 @@
-                            android:layout_width="wrap_content"
-                            android:layout_height="wrap_content"
-                            android:padding="@dimen/dp_5"
-                            android:text="@{viewModel.quantity}"
-                            android:textSize="@dimen/sp_18"
-                            />
-                        <View
-                            android:layout_width="@dimen/dp_1"
-                            android:layout_height="match_parent"
-                            android:layout_margin="@dimen/dp_5"
-                            android:background="@color/cardview_dark_background"
-                            />
-                        <ImageButton
-                            android:id="@+id/button_up_quantity"
-                            android:layout_width="wrap_content"
-                            android:layout_height="wrap_content"
-                            android:layout_gravity="center"
-                            android:background="@android:color/transparent"
-                            android:onClick="@{()-> viewModel.upQuantity()}"
-                            android:padding="@dimen/dp_5"
-                            android:src="@drawable/ic_add_black"
-                            />
-                    </LinearLayout>
+                        android:gravity="center"
+                        android:paddingTop="@dimen/dp_5"
+                        android:text="@{viewModel.orderTime}"
+                        android:textColor="@android:color/black"
+                        android:textSize="@dimen/sp_15"
+                        android:textStyle="italic"
+                        />
                 </LinearLayout>
-                <ImageButton
-                    android:id="@+id/button_delete_product"
-                    android:layout_width="wrap_content"
+                <ImageView
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:layout_gravity="center"
-                    android:layout_marginRight="@dimen/dp_20"
-                    android:background="@android:color/transparent"
-                    android:onClick="@{()-> viewModel.deleteProduct()}"
-                    app:srcCompat="@drawable/ic_delete"
+                    android:layout_weight="1.15"
+                    android:onClick="@{()-> viewModel.clickIconWarning()}"
+                    android:src="@drawable/ic_warning"
+                    android:visibility="@{viewModel.productTimeOut ? View.GONE : View.VISIBLE}"
                     />
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:layout_weight="1"
+                    android:gravity="center"
+                    android:orientation="vertical"
+                    android:paddingRight="@dimen/dp_10"
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:paddingEnd="@dimen/dp_10"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)